src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -482,6 +493,106 @@ else if ( value.toLowerCase().startsWith("chr") ) {
         return value;
      }
 
+    /**
+     * Transforms a given {@code value} from a MAF-valid value to a general-purpose value based on the given {@code key}.
+     * @param key The {@code key} off of which to base the transformation.  This key is the transformed key for output (i.e. the column name in the MAF file).
+     * @param value The {@code value} to transform from a MAF-valid value into a general-purpose value.
+     * @param referenceVersion The version of the reference used to create these annotations.
+     * @return The general-purpose equivalent of the given MAF-valid {@code value}.
+     */
+    public static String mafTransformInvert(final String key, final String value, final String referenceVersion ) {
+        switch (key) {
+            case MafOutputRendererConstants.FieldName_Variant_Classification:
+                if ( MafOutputRendererConstants.VariantClassificationMapInverse.containsKey(value)) {
+                    return MafOutputRendererConstants.VariantClassificationMapInverse.get(value);
+                }
+                break;
+
+            case MafOutputRendererConstants.FieldName_Chromosome:
+                if ( value.equals(MafOutputRendererConstants.FieldValue_Chromosome_Mito) ) {
+                    return MafOutputRendererConstants.FieldValue_Gencode_Chromosome_Mito;
+                }
+                else if (referenceVersion.toLowerCase().equals("hg19") || referenceVersion.toLowerCase().equals("b37")) {
+                    if ( value.length() <= 2 ) {
+                        if ( value.equals("1") || value.equals("2") || value.equals("3") || value.equals("4") ||
```

Comment:
You can use a static set and see if `value` is in the set.

Reply:
Fixed.